src/proj7BittingCerratoCohenEllmer/bantam/lexer/Scanner.java

```diff
@@ -225,14 +256,18 @@ private boolean isCompleteInt(String spelling) {
      * \f (form feed).
      * A string constant cannot exceed 5000 characters and cannot span multiple lines.
      *
-     * @param integer the current spelling of the token to check
-     * @return returns true if an integer
+     * @param spelling the current spelling of the token to check
+     * @return returns true if valid string
      */
     private boolean isCompleteString(String spelling) {
         if (spelling.length() > 1 && // prevent " from triggering "valid string"
                 spelling.startsWith("\"") && spelling.endsWith("\"")) {
-            //System.out.println("good start");
             if (spelling.length() <= 5000) {
+                for(String sym: nonValidSpecialStringSymbs){
```

Comment:
This simply doesn't work. The string "Hello \t There" will never register as a complete string as it is disregarded every time in the nonValidSpecialStringSymbols check.